rep-2000.rst

```diff
@@ -460,8 +465,9 @@ Middleware Implementation Support:
 |  rmw_connext_cpp         | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
 |                          |                     |               | and OpenEmbedded            |                                      |
 +--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
-| rmw_opensplice_cpp       | ADLink OpenSplice   | Tier 2        | All Platforms except Debian | All Architectures                    |
-|                          |                     |               | and OpenEmbedded            |                                      |
```

Comment:
These two OpenSplice related lines have some kind of diff. Can you please revert that since I don't think there is any intended change here.

Reply:
"ADLINK" not "ADLink" hence the change